src/widget/wtrackproperty.cpp

```diff
@@ -1,18 +1,44 @@
+#include "widget/wtrackproperty.h"
+
+// Qt includes
+#include <QMenu>
+#include <QtCore/QItemSelectionModel>
+#include <QtWidgets/QInputDialog>
+#include <QtWidgets/QCheckBox>
+#include <QtWidgets/QWidgetAction>
 
-#include <QDebug>
-#include <QUrl>
+// std includes
+#include <utility>
 
+// Project includes
 #include "control/controlobject.h"
-#include "widget/wtrackproperty.h"
+#include <library/crate/cratefeaturehelper.h>
```

Comment:
```suggestion
#include "library/crate/cratefeaturehelper.h"
```

Reply:
I will fix this in the code cleanup. This might have been added by my IDE.